Realm/RLMSyncManager.h

```diff
@@ -79,14 +79,6 @@ typedef void(^RLMSyncErrorReportingBlock)(NSError *, RLMSyncSession * _Nullable)
 @property (nonatomic, copy) NSString *appID;
 
 /**
- Whether SSL certificate validation should be disabled. SSL certificate validation is ON by default. Setting this
```

Comment:
no need to log a warning to the console if we mark the API as deprecated in the interface, leading to a compile-time warning instead.